tests/lib/dbus.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+DISPLAY=:0
+
+# shellcheck source=tests/lib/pkgdb.sh
+. "$TESTSLIB/pkgdb.sh"
+
+init_dbus_env(){
+    export $(cat dbus.env | xargs)
+}
+
+start_dbus_loop_unit(){
+    local executable="$1"
+
+    distro_install_package dbus-x11
```

Comment:
We're trying to centralize the package installations, which in this case it's also nice because we avoid the curious side-effect here. Please talk to Sergio to see how whether his branch is already in and how he organized these.